tests/Unit/TelegramTest.php

```diff
@@ -71,6 +96,53 @@ public function getBotName()
     /**
      * @test
      */
+    public function enableAdmins()
+    {
+        $tg = &$this->telegram;
+
+        $this->assertEmpty($tg->getAdminList());
+
+        $tg->enableAdmin(1);
+        $this->assertCount(1, $tg->getAdminList());
+
+        $tg->enableAdmins([2, 3]);
+        $this->assertCount(3, $tg->getAdminList());
+
+        $tg->enableAdmin(2);
+        $this->assertCount(3, $tg->getAdminList());
+
+        $tg->enableAdmin('a string?');
+        $this->assertCount(3, $tg->getAdminList());
+    }
+
+    /**
+     * @test
+     */
+    public function addCustomCommandsPaths()
```

Comment:
It's nicer, yes, because then it wouldn't need the `@test` comment above each function.

But I thought of changing just that for all tests in a single PR, OK?